seed/lib/mcm/mapper.py

```diff
@@ -99,8 +99,8 @@ def apply_column_value(raw_column_name, column_value, model, mapping, is_extra_d
     if raw_column_name in mapping:
         table_name, mapped_column_name, display_name, is_extra_data = mapping.get(raw_column_name)
 
-        # special postal case:
-        if mapped_column_name == 'postal_code':
+        # special postal case
+        if mapped_column_name in ['postal_code', 'owner_postal_code']:
             if column_value:
                 if '-' in str(column_value):
                     postal = str(column_value).split('-')[0].zfill(5)
```

Comment:
Thanks for catching the owner_postal_code.

Reply:
@nllong I recalled wrong...the reason it's not packing 0 zip with 0s is because i checked column_value here before I convert. Will change that.